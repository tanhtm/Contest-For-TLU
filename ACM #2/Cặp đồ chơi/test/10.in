100000000000000 99999999999999