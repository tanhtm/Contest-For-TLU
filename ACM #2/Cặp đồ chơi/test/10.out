49999999999999